### PR TITLE
Primeria realease para o SBrT'24

### PR DESCRIPTION
**Pull request by pertum**
# **Avaliação de Modelos para Aprimoramento de Fala Usando o Conjunto de Dados NTCD-TIMIT**

**Autores**: Augusto Cesar Becker, Gabriel Saatkamp Lazaretti, Rafael Rodrigo Pertum, Eduardo Vinícius Kuhn e Rui Seara

 **Dataset**: [NTCD-TIMIT](https://zenodo.org/record/1172064)

## **Resumo**
Este trabalho visa avaliar o desempenho das arquiteturas desenvolvidas por Park et al. [1] e Zhang et al. [2] para a tarefa de aprimoramento de sinais de fala. Especificamente, tais arquiteturas são aqui implementadas usando linguagem Python e a biblioteca TensorFlow, treinadas no mesmo conjunto de dados (público) e com hiperparâmetros adequadamente escolhidos, e os modelos obtidos são avaliados através de métricas padronizadas. Resultados de simulação indicam que ambos os modelos melhoraram a qualidade e inteligibilidade dos sinais de fala processados, independentemente da SNR e das características do ruído.

## **Referências**
[1] S. R. Park e J. Lee, A Fully Convolutional Neural Network for Speech
Enhancement, 2016. arXiv: 1609.07132 [cs.LG].

[2] X. Zhang et al., “Low-Delay Speech Enhancement Using Perceptually
Motivated Target and Loss,” em Proc. Interspeech, 2021, pp. 2826–
2830.

### CODE REVIEW

**Review by AugustoCBecker — APPROVED** [review]
Parece estar tudo certo, vamos encerrar os trabalhos para o SBrT 2024.

Muito obrigado pelo apoio!
